test/e2e/basic/backup-volume-info/csi_snapshot.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package basic
+
+import (
+	"fmt"
+
+	. "github.com/onsi/gomega"
+
+	. "github.com/vmware-tanzu/velero/test/e2e/test"
+	. "github.com/vmware-tanzu/velero/test/util/providers"
+	. "github.com/vmware-tanzu/velero/test/util/velero"
+)
+
+var CSISnapshotVolumeInfoTest func() = TestFunc(&CSISnapshotVolumeInfo{
+	BackupVolumeInfo{
+		SnapshotVolumes: true,
+		TestCase: TestCase{
+			CaseBaseName: "csi-snapshot-volumeinfo",
```

Comment:
This test case should be skipped on vSphere pipeline since we're not going test Velero CSI plugin on vSphere cluster in near future.

Reply:
Add the check in the base test case.